### PR TITLE
Xr packagify 2256

### PR DESCRIPTION
**Pull request by MichaelHazani**
adding an [XR Packages](https://github.com/webaverse/xrpackage) build script: `npm run build-xrpk`

incl. Q&As to determine manifest name, desc., type & entry point

Please test that it fully works for you!

### CODE REVIEW

File: src/engine/util/xrpk/xrpk-build.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+      ])
+      .then(answers => {
+        jsonManifest.name = answers.name;
+        jsonManifest.description = answers.description;
+        jsonManifest.xr_type = answers.xr_type;
+        jsonManifest.start_url = answers.start_url;
+        revisedJSON = JSON.stringify(jsonManifest);
+      })
+      .then(async () => {
+        await console.log("-------------------\nupdating manifest");
```
Comment (CodeRhymesLife) [inline src/engine/util/xrpk/xrpk-build.js:85]: does this console.log need an `await`?

File: src/engine/util/xrpk/xrpk-build.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        await fs.promises.writeFile(
+          "./dist/manifest.json",
+          revisedJSON,
+          err => {
+            if (err) {
+              console.error("Error writing file", err);
```
Comment (CodeRhymesLife) [inline src/engine/util/xrpk/xrpk-build.js:91]: if we fail to update the maniffest, should we delete it?

Comment (CodeRhymesLife) [reply src/engine/util/xrpk/xrpk-build.js:91]: then exit?

File: src/engine/util/xrpk/xrpk-build.js (hunk 1 above)
Comment (MichaelHazani) [reply src/engine/util/xrpk/xrpk-build.js:85]: you know, I really don't think it does:). A remnant of being in async hell for a minute there.

File: src/engine/util/xrpk/xrpk-build.js (hunk 2 above)
Comment (MichaelHazani) [reply src/engine/util/xrpk/xrpk-build.js:91]: good idea! will do

**Comment by CodeRhymesLife** [comment]
Last change looks great

**Pull request merged**